spec/workers/pull_mdm_id_worker_spec.rb

```diff
@@ -48,60 +48,60 @@
+            "Namespaced::Person(#{person.id}) has no global_registry_id; will retry"
         end
       end
 
-      context 'entity missing mdm id' do
-        let(:person) { create(:person, global_registry_id: '22527d88-3cba-11e7-b876-129bd0521531') }
+      context "entity missing mdm id" do
+        let(:person) { create(:person, global_registry_id: "22527d88-3cba-11e7-b876-129bd0521531") }
         let!(:request) do
-          stub_request(:get, 'https://backend.global-registry.org/entities/22527d88-3cba-11e7-b876-129bd0521531')
-            .with(query: { 'filters[owned_by]' => 'mdm' })
-            .to_return(body: file_fixture('get_entities_person.json'), status: 200)
+          stub_request(:get, "https://backend.global-registry.org/entities/22527d88-3cba-11e7-b876-129bd0521531")
+            .with(query: {"filters[owned_by]" => "mdm"})
+            .to_return(body: file_fixture("get_entities_person.json"), status: 200)
         end
 
-        it 'should raise an exception' do
+        it "should raise an exception" do
           expect do
             worker.pull_mdm_id_from_global_registry
           end.to raise_error GlobalRegistry::Bindings::EntityMissingMdmId,
-                             "GR entity #{person.global_registry_id} for Namespaced::Person(#{person.id}) has " \
-                             'no mdm id; will retry'
+            "GR entity #{person.global_registry_id} for Namespaced::Person(#{person.id}) has " \
+            "no mdm id; will retry"
         end
       end
 
-      context 'entity missing mdm id' do
-        let(:person) { create(:person, global_registry_id: '22527d88-3cba-11e7-b876-129bd0521531') }
+      context "entity missing mdm id" do
+        let(:person) { create(:person, global_registry_id: "22527d88-3cba-11e7-b876-129bd0521531") }
         let!(:request) do
-          stub_request(:get, 'https://backend.global-registry.org/entities/22527d88-3cba-11e7-b876-129bd0521531')
-            .with(query: { 'filters[owned_by]' => 'mdm' })
-            .to_return(body: file_fixture('get_entities_person_mdm.json'), status: 200)
+          stub_request(:get, "https://backend.global-registry.org/entities/22527d88-3cba-11e7-b876-129bd0521531")
+            .with(query: {"filters[owned_by]" => "mdm"})
+            .to_return(body: file_fixture("get_entities_person_mdm.json"), status: 200)
         end
 
-        it 'should raise an exception' do
+        it "should raise an exception" do
           expect do
             worker.pull_mdm_id_from_global_registry
-            expect(person.global_registry_mdm_id).to eq 'c81340b2-7e57-4978-b6b9-396f21bb0bb2'
+            expect(person.global_registry_mdm_id).to eq "c81340b2-7e57-4978-b6b9-396f21bb0bb2"
           end.not_to raise_error
         end
       end
     end
   end
 
-  describe '#mdm_worker_class' do
+  describe "#mdm_worker_class" do
     before do
       module MdmTest
```

Comment:
@jbirdjavi I can't figure out how to satisfy Rubocop here, but it must be similar to what I managed in spec/acceptance/global_registry_bindings_spec.rb

Reply:
References:
https://docs.rubocop.org/rubocop/cops_lint.html#lintconstantdefinitioninblock
https://relishapp.com/rspec/rspec-mocks/docs/mutating-constants/stub-undefined-constant
https://relishapp.com/rspec/rspec-mocks/docs/basics/scope

Reply:
I think this is one case where defining within a block should be okay? I would think everything with a `before` block gets carried over to the actual specs. I'd probably just disable the cop here.

Reply:
I found some examples in `sidekiq-unique-jobs/spec` that might help